servers/display_server.cpp

```diff
@@ -698,6 +702,7 @@ void DisplayServer::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("screen_get_scale", "screen"), &DisplayServer::screen_get_scale, DEFVAL(SCREEN_OF_MAIN_WINDOW));
 	ClassDB::bind_method(D_METHOD("is_touchscreen_available"), &DisplayServer::is_touchscreen_available, DEFVAL(SCREEN_OF_MAIN_WINDOW));
 	ClassDB::bind_method(D_METHOD("screen_get_max_scale"), &DisplayServer::screen_get_max_scale);
+	ClassDB::bind_method(D_METHOD("ios_set_ui_state", "hide_home_indicator", "suppress_bottom_ui_gesture", "suppress_top_ui_gesture", "hide_status_bar", "status_bar_fade_time"), &DisplayServer::ios_set_ui_state, DEFVAL(true), DEFVAL(0.2));
```

Comment:
Having a separate `get/set` methods for each value is probably better.

Reply:
Also, documentation should be updated after adding new methods, see [Updating class reference when working on the engine](https://docs.godotengine.org/en/latest/contributing/documentation/updating_the_class_reference.html#updating-class-reference-when-working-on-the-engine).